go.mod

```diff
@@ -1,111 +1,110 @@
 module github.com/athanorlabs/atomic-swap
 
-go 1.20
+go 1.21
 
 require (
 	filippo.io/edwards25519 v1.0.0
 	github.com/ChainSafe/chaindb v0.1.6
 	github.com/MarinX/monerorpc v1.0.7
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/athanorlabs/go-dleq v0.1.0
-	github.com/athanorlabs/go-p2p-net v0.3.0
+	github.com/athanorlabs/go-p2p-net v0.3.1-0.20231210033046-13ef86a8318f
```

Comment:
Will update to a tagged release before merging.

Reply:
approved the go-p2p-net PR!